### PR TITLE
add code to blur the license plates

### PR DESCRIPTION
**Pull request by CalebMuhia**

### CODE REVIEW

File: plate_recognition.py [hunk 1]
```diff
@@ -58,6 +78,25 @@ def main():
                         time.sleep(1)
                     else:
                         break
+            if args.blur_amount:
+                for res in response.json()['results']:
+                    box = res['box']
+
+                    im = Image.open(path)
+                    mask = Image.new('L', im.size, 0)
```
Comment (marcbelmont) [inline plate_recognition.py]: Can you use this instead? It's cleaner.

https://stackoverflow.com/questions/9701515/filter-part-of-image-using-pil-python

File: README.md [hunk 2]
```diff
@@ -65,6 +65,15 @@ You can also run the license plate reader on many files at once. To run the scri
 
 `python plate_recognition.py --api-key MY_API_KEY /path/to/car1.jpg /path/to/car2.jpg /path/to/trucks*.jpg`
 
+### Bluring the license plate
```
Comment (marcbelmont) [inline README.md]: Spelling: Blurring License Plates

File: README.md [hunk 3]
```diff
@@ -65,6 +65,15 @@ You can also run the license plate reader on many files at once. To run the scri
...
+
+You can also blur the license plate with the following parameters: `--blur-amount AMOUNT`. `AMOUNT` is a number between 0 and 50. you should also supply a directory to save the blurred images. use:
```
Comment (marcbelmont) [inline README.md]: Update table of contents

File: plate_recognition.py (hunk 1 above)
Comment (CalebMuhia) [reply plate_recognition.py]: Done

File: README.md (hunk 2 above)
Comment (CalebMuhia) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (CalebMuhia) [reply README.md]: Done

File: README.md [hunk 4]
```diff
@@ -12,6 +12,7 @@ Integrate with our ALPR API in a few lines of code. Get an easy to use JSON resp
     - [Lookups For a Specific Region](#lookups-for-a-specific-region)
     - [Process Multiple Files (Batch Mode)](#process-multiple-files-batch-mode)
     - [Running the ALPR Locally (SDK)](#running-the-alpr-locally-sdk)
+    - [Blurring the license plate](#blurring-the-license-plate)
```
Comment (marcbelmont) [inline README.md]: Nit: fix casing. Blurring the License Plate

**Pull request merged**